### DISCUSSION TITLE
Struggling with setting up stations.

### DISCUSSION THREAD

**Discussion by MxHrv** [Opinionated-Blueprints/10-Books-Full-of-Rails#127] [Q&A]
I am pretty new to circuits let alone LTNs and am trying to set up a base with these stations but I am always getting a "Destination Stop full" from the train when I am trying to refuel it. There are currently no other trains in the network as I am trying to set up the first one.  Would anyone know what I am doing wrong?

**Comment by bwoods89** [comment]
Are the fuel supply and refueling stations properly setup? Is the refueling train setup?
